platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitProjectType.java

```diff
@@ -21,12 +21,18 @@
 @Singleton
 public class GitProjectType extends TransientMixin {
 
-    public static final String VCS_PROVIDER_NAME = "vcs.provider.name";
+    public static final String VCS_PROVIDER_NAME   = "vcs.provider.name";
+    public static final String CURRENT_BRANCH_NAME = "current.branch.name";
```

Comment:
Should we use commons name of these variables for all vcs? If yes maybe it would be better to add vcs. as prefix of attributes name. If no please think about adding of git. prefix to names of these (CURRENT_BRANCH_NAME and REPOSITORY_REMOTES) attributes